src/main/resources/test_1.py

```diff
@@ -0,0 +1,3 @@
+a = 5
```

Comment:
Sorry, but I think you should use the test's resources folder to store tests files